package.json

```diff
@@ -40,6 +40,7 @@
     "mocha": "^7.1.2",
     "node-fetch": "^2.6.0",
     "nyc": "^15.0.1",
+    "rimraf": "^3.0.2",
```

Comment:
This is one of those `left-pad` level dependencies where we need a module for `rm -rf` since it's still [experimental](https://nodejs.org/api/fs.html#fs_fs_rmdir_path_options_callback) in Node's standard APIs still :1st_place_medal: 

Luckily it's just a dev dependency...

Reply:
Yes, I'm using that a couple places myself in my build scripts.